metaflow/client/core.py

```diff
@@ -337,17 +350,17 @@ def _get_object(self, *path_components):
             raise MetaflowNotFound("%s does not exist" % self)
         return result
 
-    def __iter__(self) -> Iterable["MetaflowObject"]:
+    def __iter__(self) -> Iterator["MetaflowObject"]:
```

Comment:
I changed all things that have yield to `Yields` in the docstring and also to return an Iterator (proper return type for yields).